src/util.ts

```diff
@@ -161,3 +161,6 @@ export const removeFrontMatterFromContent = (content: string): string => {
 
   return content.replace(frontMatterRegex, "");
 };
+
+export const snakeToCamelCase = (str: string) =>
+  str.replace(/(_[a-z])/g, (group) => group.toUpperCase().replace("_", ""));
```

Comment:
Does this mean numbers and `/` would be replaced?

Reply:
Oh nevermind, this is for variables, i thought it was for escaping label names.

Reply:
no, i think it only converts `_` following with a lower case character into a upper case one.
For example, `_a` -> `A`